routes/files/create-update.js

```diff
@@ -75,7 +75,13 @@ module.exports = function(config, req, res) {
       });
 
       response.on('end', function() {
-        body = JSON.parse(body);
+        try {
+          body = JSON.parse(body);
+        } catch(e) {
+          console.error("Failed to parse response for sending file with ", e.message, "\n at ", e.stack);
+          res.sendStatus(500);
```

Comment:
So the `utils.error` thing seems to be specific only to middleware and I think it wraps the error in webmaker stuff! I think we'll need to figure that out when we refactor the server and remove old code (I believe we won't need to do any of that `utils` stuff then).